flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/data/TestFlinkOrcReaderWriter.java

```diff
@@ -104,4 +110,10 @@ protected void writeAndValidate(Schema schema) throws IOException {
       assertThat(records).isExhausted();
     }
   }
+
+  @Override
+  @Disabled("ORC file format supports null values even for required fields")
```

Comment:
similarly if we should just modify the test code behavior (instead of ignore/disable).